Python/_warnings.c

```diff
@@ -1031,28 +1031,31 @@ get_source_line(PyInterpreterState *interp, PyObject *module_globals, int lineno
     return source_line;
 }
 
+/*[clinic input]
+warn_explicit as warnings_warn_explicit
+
+    message: object
+    category: object
+    filename: unicode
+    lineno: int
+    mod: object = NULL
```

Comment:
```suggestion
    module: object = NULL
```

Reply:
@erlend-aasland The `module` parameter is reserved by Argument Clinic and points to a parent module object:

```cpp
static PyObject *
warnings_warn_explicit_impl(PyObject *module, PyObject *message,
                            PyObject *category, PyObject *filename,
                            int lineno, PyObject *mod, PyObject *registry,
                            PyObject *module_globals, PyObject *sourceobj)
```

Reply:
Then we should fix that first. This PR is currently a change in behaviour; one keyword argument is now named differently than before.

Reply:
I believe that a risk to break anything is next to zero:

- `warnings.warn_explicit` [performs lots of useful checks](https://github.com/python/cpython/blob/main/Lib/warnings.py#L327-L395) before calling `_warnings.warn_explicit` so users have no benefit to plug into the private implementation directly,
- the _warnings module is a private, undocumented backend.

> Then we should fix that first

The problem is not creation of the issue. The problem is that it'll stuck because the module arg renaming feature brings no use outside of a single PR.